sdk/python/tests/compiler/testdata/condition.log

```diff
@@ -0,0 +1,8 @@
+[flip : main] heads
+
+[flip-again : main] tails
+
+[print1 : main] tails
+[print1 : main]
+
+Unable to get Taskrun: taskruns.tekton.dev "flip-coin-example-pipeline-run-print2-c7766" not found
```

Comment:
This is probably a bug from the tkn cli, the cli shouldn't try to log when the condition is failed.

Reply:
Yup, update it after a couple more successful runs without that error message